assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -15,10 +17,13 @@ import { useCovers } from './useCovers';
 import { getStyleFromClassName } from '../getStyleFromClassName';
 import { CoversGridLoadMoreButton } from './CoversGridLoadMoreButton';
 import { CoversCarouselLayout } from './CoversCarouselLayout';
+import withCharacterCounter from "../../components/withCharacterCounter/withCharacterCounter";
 
 const { RichText } = wp.blockEditor;
 const { __ } = wp.i18n;
 
+const TextControl = withCharacterCounter(BaseTextControl);
```

Comment:
This is not useful at the moment, since no character limit was set it shows as a regular text input without any counter